update/genesis/import.go

```diff
@@ -22,6 +22,8 @@ import (
 
 var _ update.ImportHandler = (*stateImport)(nil)
 
+const maxTrieLevelInMemory = uint(5)
```

Comment:
why you dont use value from cofig file ?

Reply:
Because it is not needed. The tries that are created here are used just to import the data, so there is no need to pass the config.